packages/toolpad-app/src/toolpadComponents/index.tsx

```diff
@@ -3,7 +3,9 @@ import * as appDom from '../appDom';
 
 export interface ToolpadComponentDefinition {
   displayName: string;
-  builtin?: string;
+  builtIn?: string;
+  display: boolean;
```

Comment:
Let's make that `system: true` for the few system components and omit the property for the others. We're most likely gonna remove these from this array in the future and define them directly in the runtime.

```suggestion
  system?: boolean;
```